Resources/Prototypes/Entities/Objects/Shields/shields.yml

```diff
@@ -400,3 +400,65 @@
       sprite: Objects/Weapons/Melee/e_shield.rsi
       size: 5
       heldPrefix: eshield
+
+- type: entity
+  name: telescopic shield
+  parent: BaseShield
+  id: TelescopicShield
+  description: An advanced riot shield made of lightweight materials that collapses for easy storage.
+  components:
+    - type: ItemToggle
+      activatedDisarmMalus: 0.6
+      activateSound: false
+      deActivateSound: false
```

Comment:
Add a sound or remove this.

Reply:
If you do not install these components, the Client crashes when trying to look at the shield. I'll try to find the sound then.